docs/api.md

```diff
@@ -3533,4 +3534,4 @@ TimeoutError is emitted whenever certain operations are terminated due to timeou
 [Worker]: #class-worker "Worker"
 [Accessibility]: #class-accessibility "Accessibility"
 [AXNode]: #accessibilitysnapshotoptions "AXNode"
-[ConnectionTransport]: ../lib/WebSocketTransport.js "ConnectionTransport"
\ No newline at end of file
+[ConnectionTransport]: ../lib/WebSocketTransport.js "ConnectionTransport"
```

Comment:
a final new line was added at the end of this file, conforming with `.editorconfig`